### PR TITLE
Raise PHPStan level to 9

### PR DESCRIPTION
**Pull request by clxmstaab**
raise to level 9 for max safety.

I tried locally reverting the fix from https://github.com/ondrejmirtes/simple-downgrader/commit/6a8d0f0c1a4fae734f03161d3d19d7437058ca0b but level 9 still does not catch it

refs https://github.com/ondrejmirtes/simple-downgrader/pull/3/files#r1444030185

### CODE REVIEW

**Comment by clxmstaab** [comment]
with `checkImplicitMixed: true` it would be catched

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]
1) Try checkImplicitMixed too
2) Try fixing the errors instead of the baseline

**Comment by staabm** [comment]
@ondrejmirtes done

File: phpstan-baseline.neon [hunk 1]
```diff
@@ -0,0 +1,6 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Method SimpleDowngrader\\\\PhpDoc\\\\CallbackVisitor\\:\\:enterNode\\(\\) should return 1\\|2\\|3\\|4\\|array\\<PHPStan\\\\PhpDocParser\\\\Ast\\\\Node\\>\\|PHPStan\\\\PhpDocParser\\\\Ast\\\\Node\\|null but returns mixed\\.$#"
```
Comment (staabm) [inline phpstan-baseline.neon]: see https://github.com/phpstan/phpstan/issues/10409

**Review by ondrejmirtes — APPROVED** [review]

**Comment by ondrejmirtes** [comment]
Thank you.

**Pull request merged**